src/components/ToolMenu/LayerTree/LayerTreeContextMenu/index.tsx

```diff
@@ -293,6 +316,18 @@ export const LayerTreeContextMenu: React.FC<LayerTreeContextMenuProps> = ({
     });
   }
 
+  if (isGeoserverLayer) {
+    dropdownMenuItems.push({
+      label: 'Geostyler',
```

Comment:
The label should be translatable and the current one might be confusing to users. We might want to replace it with something like `Layer style`.